mule-migration-tool-e2e-tests/src/test/java/com/mulesoft/tools/migration/e2e/AbstractEndToEndTestCase.java

```diff
@@ -160,20 +170,57 @@ private void comparePom(Path expectedPath, Path migratedPath) {
     }
   }
 
+  private void normalizeVersions(PomModel pomModel) {
+    normalizeVersionProperty(pomModel, MULE_TOOLS_VERSION_POM_PROPERTY);
+    normalizeVersionProperty(pomModel, MULE_VERSION_POM_PROPERTY);
+    pomModel.getDependencies().forEach(dep -> {
+      if (dep.getVersion() != null && VERSION_REGEX_MATCHER.matcher(dep.getVersion()).matches()) {
+        dep.setVersion(VERSION_PLACEHOLDER);
+      }
+    });
+    pomModel.getPlugins().forEach(plugin -> {
+      if (plugin.getVersion() != null && VERSION_REGEX_MATCHER.matcher(plugin.getVersion()).matches()) {
+        plugin.setVersion(VERSION_PLACEHOLDER);
+      }
+    });
+  }
+
+  private void normalizeVersionProperty(PomModel pomModel, String propertyName) {
+    if (pomModel.getProperties().getProperty(propertyName) != null) {
+      pomModel.getProperties().setProperty(propertyName, VERSION_PLACEHOLDER);
+    }
+  }
+
   private void compareJson(Path expected, Path migratedPath) {
     try {
       JsonElement expectedJson = JsonParser.parseString(IOUtils.toString(expected.toUri(), UTF_8));
       JsonElement actualJson = JsonParser.parseString(IOUtils.toString(migratedPath.toUri(), UTF_8));
       if (migratedPath.getFileName().endsWith("report.json")) {
         normalizeFilePath(expectedJson);
         normalizeFilePath(actualJson);
+        // connector names have versions included [0-9]{1,2}\.[0-10]{1,2}\.[0-10]{1,2}
       }
+
+      removeVersions(actualJson.getAsJsonObject().get("connectorsMigrated").getAsJsonArray());
```

Comment:
same as with pom, I'd run the normalizer on the expected json report too, so there's no need to update the actual report